src/scene/materials/standard-material.js

```diff
@@ -298,10 +299,10 @@ const _tempColor = new Color();
  * within the medium. Only used when useDynamicRefraction is enabled.
  * @property {Color} emissive The emissive color of the material. This color value is 3-component
  * (RGB), where each component is between 0 and 1.
- * @property {boolean} emissiveTint Multiply emissive map and/or emissive vertex color by the
- * constant emissive value.
- * @property {Texture|null} emissiveMap The emissive map of the material (default is null). Can be
- * HDR.
+ * @property {Texture|null} emissiveMap The emissive
+ * map of the material (default is null). Can be HDR. When the emissive map is applied, the emissive
+ * color is multiplied by the texel color in the map. Since the emissive color is black by default,
+ * the emissive map won't be visible unless the emissive color is changed.
```

Comment:
```suggestion
* @property {Texture|null} emissiveMap The emissive map of the material (default is null). Can be
* HDR. When the emissive map is applied, the emissive color is multiplied by the texel color in the
* map. Since the emissive color is black by default, the emissive map won't be visible unless the
* emissive color is changed.
```